rust/signed_doc/src/metadata/mod.rs

```diff
@@ -464,53 +253,21 @@ impl minicbor::Decode<'_, crate::decode_context::DecodeContext<'_>> for Metadata
     fn decode(
         d: &mut Decoder<'_>, ctx: &mut crate::decode_context::DecodeContext<'_>,
     ) -> Result<Self, minicbor::decode::Error> {
-        const REPORT_CONTEXT: &str = "Metadata decoding";
+        // TODO: use helpers from `cbork-utils` crate to verify that's the map is
+        // deterministically CBOR encoded map.
```

Comment:
```suggestion
        if let Err(err) = cbork_utils::deterministic_helper::decode_map_deterministically(&mut d.probe()) {
            ctx.report.other(&err.to_string(), "Metadata deterministic encoding validation");
        }
```
This silently checks as per `TODO`. The rest of the decoding logic is unaffected.